ui-lib/src/components/Message/Alert/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import Text from "@components/Text";
+import ShieldSecurityMedium from "@ui/assets/icons/ShieldSecurityMedium";
+import CircledCrossMedium from "@ui/assets/icons/CircledCrossMedium";
+import CircledAlertMedium from "@ui/assets/icons/CircledAlertMedium";
+
+type AlertType = "info" | "warning" | "error";
+
+export interface AlertProps {
+  type?: AlertType;
+  title: string;
+  showIcon?: boolean;
+}
+
+const StyledIconContainer = styled.div`
+  margin-right: 12px;
+  display: flex;
+  align-items: center;
+`;
+
+const getIcon = (type: AlertType) => {
+  switch (type) {
+    case "info":
+      return (
+        <StyledIconContainer>
```

Comment:
you're repeating the container quite a bit why not put it in the template like 
```
<StyledIconContainer>
 { getIcon(type) }
</StyledIconContainer>
```

Reply:
also i tend to prefer the indexer notation than a switch like `icons[type]` to access a specific value :grimacing:

Reply:
I just made the recommanded change, it's simpler and clearer :ok_hand: